### ISSUE TITLE
"SKIP_MODULES" is not present in the VALID_SETTINGS in 'rest_framework_roles/parsing.py'

### ISSUE THREAD

**Issue by Fa2y** [Pithikos/rest-framework-roles#5]
As the title suggests, the "SKIP_MODULES" is not present in the VALID_SETTINGS, so when specified in the settings it will throw the following error:
```
...
  File "/usr/local/lib/python3.9/site-packages/rest_framework_roles/parsing.py", line 89, in parse_view_permissions
    roles = load_roles()
  File "/usr/local/lib/python3.9/site-packages/rest_framework_roles/parsing.py", line 36, in load_roles
    settings = load_settings(config)
  File "/usr/local/lib/python3.9/site-packages/rest_framework_roles/parsing.py", line 28, in load_settings
    validate_config(config)
  File "/usr/local/lib/python3.9/site-packages/rest_framework_roles/parsing.py", line 18, in validate_config
    raise django_exceptions.ImproperlyConfigured(f"Unknown setting '{setting}'")
django.core.exceptions.ImproperlyConfigured: Unknown setting 'SKIP_MODULES'
```

**Comment by Fa2y** [comment]
I've submitted a pull request that resolve it.

**Comment by Pithikos** [comment]
Thanks for the hotfix. It's included in v1.0.3 now :+1:
